tests/end-to-end/external-solution-integration/internal/scenario/compass_e2e/compass_e2e.go

```diff
@@ -27,6 +27,7 @@ func (s *CompassE2EScenario) Steps(config *rest.Config) ([]step.Step, error) {
 	compassClients := testkit.InitCompassClients(kymaClients, state, s.domain, s.skipSSLVerify)
 	knativeEventingClientSet := eventing.NewForConfigOrDie(config)
 	serviceBindingUsageClientset := serviceBindingUsageClient.NewForConfigOrDie(config)
+	httpSourceClientset := sourcesclientv1alpha1.NewForConfigOrDie(config)
```

Comment:
nitpick: iPlease be consistent and make it "Clientset" everywhere . see lines above